src/helper.cc

```diff
@@ -576,6 +574,12 @@ helper::submit(const char *buf, HLPCB * callback, void *data)
     debugs(84, DBG_DATA, Raw("buf", buf, strlen(buf)));
 }
 
+void
+helper::callBack(HLPCB *callback, void *data, const Helper::Reply &reply)
```

Comment:
Most likely, this method should take just Xaction or Request+Reply parameters, not raw data.

Reply:
Without SubmissionFailure() changes, we do not need these parameters of course. In PR229, callBack() takes a single Xaction * argument.